src/utils/options.ts

```diff
@@ -0,0 +1,107 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import * as os from 'os';
+
+import { ExpirationStrategy } from './cache/expiration-strategy';
+import { MemoryStorage } from './cache/memory-storage';
+
+export class Options {
+  private configFile: string;
+
+  private logFile: string;
+
+  private readonly cache: ExpirationStrategy;
+
+  constructor() {
+    this.cache = new ExpirationStrategy(new MemoryStorage());
+    const wakaHome = this.getWakaHome();
+    this.configFile = path.join(wakaHome, '.wakatime.cfg');
+    this.logFile = path.join(wakaHome, '.wakatime.log');
+  }
+
+  private getWakaHome(): string {
+    const home = process.env.WAKATIME_HOME;
```

Comment:
Sometimes WAKATIME_HOME variable is set using the symbol `~` and/or with trailing slash at the end like `~/mypath` or `~/mypath/`. It needs to be expanded before returning. Does NodeJS resolve "~"?

Reply:
if you go to the `constructor` when we use the `wakaHome` on path.join node normalizes the path so the situation is been handle there 😎